t/007-clear-cache.t

```diff
@@ -0,0 +1,73 @@
+#!perl
+
+use strict;
+use warnings FATAL => 'all';
+
+use Test::More tests => 14;
+
+use constant {
+    ADDRESS => 'wikipedia.org',
+    DIFFERENT_ADDRESS => 'en.wikipedia.org',
+    SCHEME => 'https://',
+};
+
+use Test::RequiresInternet ( (ADDRESS) => 443 );
```

Comment:
I think this could be more specific?

`use Test::RequiresInternet ( 'en.wikipedia.org' => 443, 'www.wikipedia.org' => 443 );`